Sources/SwiftInspectorVisitors/TypeDescription.swift

```diff
@@ -247,18 +278,12 @@ extension TypeSyntax {
       return .implicitlyUnwrappedOptional(typeIdentifier.wrappedType.typeDescription)
 
     } else if let typeIdentifier = self.as(ArrayTypeSyntax.self) {
-      // An array of the form [Type] is synonymous with Array<Type>.
-      // Utilize a simple type rather than creating another enum case.
-      let elementDescription = typeIdentifier.elementType.typeDescription.asSource
-      return .simple(name: "Array<\(elementDescription)>")
+      return .array(element: typeIdentifier.elementType.typeDescription)
```

Comment:
Another possible option here is to encode this type as: 
```suggestion
      return .simple(name: "Array", generics: [typeIdentifier.elementType.typeDescription])
```

Reply:
I actually prefer the .array case tbh.

Reply:
I guess we can always have consuming code make this mapping if we really want it. 👍

Reply:
I know I was involved in discussions about this previously and I can't recall exactly where I stood. Moving forward I do agree we should have the `.array` and `.dictionary` cases and I also do think having a way to normalize at a higher level could be cool.